Configuration/AlCa/python/autoAlca.py

```diff
@@ -39,6 +39,47 @@
                   "StreamExpressHI" : "SiStripCalZeroBias+TkAlMinBiasHI+SiStripPCLHistos+SiStripCalMinBias+SiStripCalMinBiasAAG+SiPixelCalZeroBias"
                   }
 
+# AlCaReco matrix used in CMSSW releases in 2018
+AlCaRecoMatrix2018 = {"AlCaLumiPixels" : "AlCaPCCZeroBias+AlCaPCCRandom",
```

Comment:
Wondering if instead of copying the `dict` over and over again one could just start from the default one and update the keys that differ

Reply:
I agree Marco! But since Tamas is on holiday this week (and was kind enough to open the PR) I could do it in a follow-up PR if you agree?

Reply:
Sure, you are the one signing off anyway, merely suggesting here

Reply:
Thanks @francescobrivio for following-up on this!